src/libstd/io/buffered.rs

```diff
@@ -21,7 +21,7 @@ use io::{self, DEFAULT_BUF_SIZE, Error, ErrorKind, SeekFrom};
 use ptr;
 use iter;
 
-/// Wraps a `Read` and buffers input from it
+/// Wraps a `Read` and buffers input from it.
```

Comment:
Convention is to use full sentences, actually, so I'd prefer to leave this (and others)